README.md

```diff
@@ -249,6 +249,8 @@ lintersConfig:
     preferredRequiredMarker: required | kubebuilder:validation:Required # The preferred required marker to use, fixes will suggest to use this marker. Defaults to `required`.
 ```
 
+The `optionalorrequired` linter also checks whether defined type has optional marker or required one.
+
```

Comment:
This should not be in the configuration section of the documentation, please move above line 243

And I'd suggest something like
```suggestion
The `optionalorrequired` linter also checks for the presence of optional or required markers on type declarations, and forbids this pattern.
```

Reply:
There should also be a `doc.go` file that should be updated for this change as well